languages/python/python.go

```diff
@@ -23,7 +39,25 @@ func (l language) GetLanguage() *sitter.Language {
 }
 
 func (l language) Imports(content []byte, n *sitter.Node) ([]string, error) {
-	log.Println(n)
+	once.Do(func() { q, qerr = sitter.NewQuery([]byte(query), l.GetLanguage()) })
+	if qerr != nil {
+		return nil, qerr
+	}
+
 	var out []string
+	c := sitter.NewQueryCursor()
+	c.Exec(q, n)
+	for {
+		m, ok := c.NextMatch()
```

Comment:
if we only check the first match, then everything works fine - also the corner case:
```
from a.b import x as b
```